autodoc-admin-api/data.lua

```diff
@@ -189,7 +191,78 @@ return {
           ]],
         },
       }
-    }
+    },
+    config = {
+      title = [[Db-less configuration]],
+      description = "",
+      ["/"] = {
+        POST = {
+          title = [[Update db-less config]],
+          endpoint = [[<div class="endpoint post">/config</div>]],
+          description = [[
+            This endpoint allows resetting a db-less Kong with a new
+            yml data file.
+
+            To learn more about the yaml file needed, please run:
```

Comment:
ditto here — then later we make it a link to the yet-unwritten declarative config docs.